modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/ResolverSetUtilsTestCase.java

```diff
@@ -244,12 +247,57 @@ public Class<String> getType() {
     }
   };
 
+  private static final String CONTENT_PARAMETER_NAME = "content";
+  private static final String JSON_STREAM_OPERATION_NAME = "getStreamContentWithFlackyConnection";
+  private static final String XML_STREAM_OPERATION_NAME = "getXmlStreamContentWithFlackyConnection";
+  private static final String JSON_STRING_OPERATION_NAME = "getStringContentWithFlackyConnection";
+  private static final String XML_STRING_OPERATION_NAME = "getXmlStringContentWithFlackyConnection";
+
+  private static final Consumer<ValueDeclarer> JSON_PERSON_DECLARER = valueDeclarer -> valueDeclarer
+      .objectValue(objectValueDeclarer -> objectValueDeclarer
+          .withField("firstName", "John")
+          .withField("lastName", "Doe"));
+
+  private static final String JSON_PERSON_VALUE = "{\n  \"firstName\": \"John\",\n  \"lastName\": \"Doe\"\n}";
```

Comment:
take from file and use a better exampe.. something that has complex objects, arrays of scalar, arrays of objects, arrays of arrays, maps, etc